python/sglang/bench_offline_throughput.py

```diff
@@ -227,7 +227,7 @@ def throughput_test_once(
             "SGLANG_TORCH_PROFILER_DIR" in os.environ
         ), "Please set SGLANG_TORCH_PROFILER_DIR."
         os.makedirs(os.environ["SGLANG_TORCH_PROFILER_DIR"], exist_ok=True)
-        backend.start_profile()
+        backend.start_profile(activities=["CPU", "HPU"])
```

Comment:
it is only necessary when _is_hip is true

Reply:
fixed